### ISSUE TITLE
Do you need an English-Spanish translator?

### ISSUE THREAD

**Issue by chicooou** [crypto101/book#348]
I've been watching the interesting project, and I think I could contribute by translating from English into Spanish, which seems to them the idea: D

**Comment by guilhermeasg** [comment]
I've been thinking about translations on this too! Count me in for Portuguese or Spanish, in case the maintainers are interested. ;)

**Comment by lvh** [comment]
That's a really interesting, and not something I've thought closely about. Given the amount that Crypto 101 changes, do you have any suggestions for how to manage that? I.e. making sure that both documents eventually stay up to date? Is git diffs enough for that?

**Comment by guilhermeasg** [comment]
Can't really tell that from experience, as I've never done translations in _living documents_ before. That said, given TeX diffs are fairly readable, I wouldn't expect a lot of difficulty there.

With no other "versioning" of the book edition, we could fork the project for now and make a PR with language subdirectories. Our translated version could then claim to be up-to-date as of `920c76102fe6afee5f3f70c85c3a9d511abc11c2`, and we could add periodic (monthly?) revisions on top of that. Does that sound too hacky?

(sorry for hijacking your issue @chicooou, feel free to disagree/kick me out of the discussion)

**Comment by lvh** [comment]
Yeah; anchoring it to a git revision seems like the only way to make it work. I imagine a lot of the tweaks won't be relevant to translations; e.g. if a sentence is ambiguous in English it's possible that the translation is not.

**Comment by multun** [comment]
@chicooou you should now be able to translate crypto101 easily, see #372
